modules/apps/portal-search/portal-search-test/src/test/java/com/liferay/portal/search/internal/result/SearchResultUtilTest.java

```diff
@@ -55,6 +59,17 @@ public class SearchResultUtilTest extends BaseSearchResultUtilTestCase {
 	public static final LiferayUnitTestRule liferayUnitTestRule =
 		LiferayUnitTestRule.INSTANCE;
 
+	@Before
+	@Override
+	public void setUp() throws Exception {
+		super.setUp();
```

Comment:
@W2jx Why do you need to add a `setUp` method here and just call `super.setUp()`, without any additional logic? If you don't override it, I think the super class' setUp method can be directly called by JUnit.

Reply:
Hi @dantewang , Because I think it makes it clear that _searchResultManagerImpl is created within the logic in method setUp, and this will make it clearer why _searchResultManagerImpl.deactivate is executed in method tearDown. If you think this is redundant, I can delete it.

Reply:
@W2jx Please delete it.